src/Int.mo

```diff
@@ -15,20 +20,16 @@ module {
   /// Returns the absolute value of `x`.
   ///
   /// Example:
-  /// ```motoko
-  /// import Int "mo:base/Int";
-  ///
+  /// ```motoko include=import
   /// Int.abs(-12) // => 12
   /// ```
   public let abs : (x : Int) -> Nat = Prim.abs;
```

Comment:
Using the `func` keyword instead of a `let` generates docs that makes it more clear that the function is a function.